src/main/java/cn/nukkit/block/BlockDispenser.java

```diff
@@ -209,7 +209,7 @@ public void dispense() {
             pk.evid = LevelEventPacket.EVENT_SOUND_CLICK;
             pk.data = 1000;
 
-            this.level.addChunkPacket(getChunkX(), getChunkZ(), pk.clone());
+            //this.level.addChunkPacket(getChunkX(), getChunkZ(), pk.clone());
```

Comment:
Why did you comment this?

Reply:
Commented it since everytime it sent, the client would disconnect

Reply:
I do believe it's fine for now because it's only a sound packet